modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp

```diff
@@ -173,44 +173,66 @@ renderResponse.setTitle(blogsEditEntryDisplayContext.getPageTitle(resourceBundle
 						/>
 					</aui:fieldset>
 
-					<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="configuration">
+					<%
+					Portlet portlet = PortletLocalServiceUtil.getPortletById(BlogsPortletKeys.BLOGS);
+					%>
 
-						<%
-						Portlet portlet = PortletLocalServiceUtil.getPortletById(BlogsPortletKeys.BLOGS);
-						%>
+					<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPD-11147") %>'>
+						<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="friendly-url">
+							<div class="clearfix form-group">
+								<react:component
+									module="{AssetCategoriesFriendlyUrlSelector} from blogs-web"
+									props='<%=
+										HashMapBuilder.<String, Object>put(
+											"id", liferayPortletResponse.getNamespace() + "friendly_url_category_selector"
+										).put(
+											"inputAddon", StringUtil.shorten("/-/" + portlet.getFriendlyURLMapping(), 40) + StringPool.SLASH
```

Comment:
Why short it to 40 @NemethNorbert?

Reply:
That's a good question, I actually took that pattern from the FriendlyUrl taglib usage, However in this case, we should probably remove that cap.

Reply:
I made the changes @jkappler